pcawg_metadata_parser/parse_gnos_xml.py

```diff
@@ -192,7 +192,7 @@ def process_gnos_analysis(gnos_analysis, donors, vcf_entries, es_index, es, bam_
      
 
     # only do the following when it is WGS
-    if bam_file.get('library_strategy') == 'WGS':
+    if bam_file.get('library_strategy') == 'WGS' and bam_file.get('bam_type') == 'Specimen level aligned BAM':
```

Comment:
This is ok because we now have all unaligned BAMs aligned, using only aligned BAM entries we can populated this section without missing any WGS specimens (including normal and tumour).